templates/p5_gfd.bt

```diff
@@ -2912,6 +2941,130 @@ typedef struct( u32 version )
 // -- Model pack types --
 //
 
+//
+// == Physics Data Chunk type stuff --
+//
+typedef struct{
+	u32 unk;
+	f32 unk;
+	f32 unk;
+	f32 unk;
+	f32 unk;
+	u32 entry1count;
+	u32 entry2count;
+	u32 entry3count;
+} TChunkTypePhysicsDictionaryHeader <name="PhysicsDictionaryHeader">;
+
+typedef struct{
+	f32 field34;
+	f32 field38;
+	f32 field3C;
+	f32 field40;
+	byte hasName;
+	if (hasName > 0)
+	{
+		byte unk<hidden = true>;
+		byte nameSize;
+		char boneName[nameSize];
+		u32 stringHash<format=hex>;
+	}
+	else{
+		f32 field18;
+		f32 field14;
+		f32 field10;
+	}
+	
+}TEntryType1<name = "Type 1 Entry">;
+
+typedef struct{
+	u16 Field94;
+	f32 field84;
+	if (Field94 == 1)
+	{
+		f32 Field88;
+	}
+	TMatrix4x4 Field8C;
+	byte hasName;
+	if (hasName > 0)
+	{
+		byte unk<hidden = true>;
+		byte nameSize;
+		char boneName[nameSize];
+		u32 stringHash<format=hex>;
+	}
+}TEntryType2<name = "Type 2 Entry">;
+
+typedef struct{
+	f32 Field00;
+	f32 Field04;
+	
+	f32 field08;
+	u16 parentBone;
+	u16 childBone;
+}TEntryType3<name = "Type 3 Entry">;
+
+typedef struct(u32 count) {
+	local u32 i;
+	for (i = 0; i < count; i++)
+	{
+		TEntryType1 entryType1<read=NameToStringT1>;
+	}
+}TEntryType1Wrapper;
+
+typedef struct(u32 count) {
+	local u32 i;
+	for (i = 0; i < count; i++)
+	{
+		TEntryType2 entryType2<read=NameToStringT2>;
+	}
+}TEntryType2Wrapper;
+
+typedef struct(u32 count) {
+	local u32 i;
+	for (i = 0; i < count; i++)
+	{
+		TEntryType3 entryType3;
+	}
+}TEntryType3Wrapper;
+
+string NameToStringT1( TEntryType1& value )
+{
+    string buffer;
+	if (value.hasName > 0)
+	{
+		buffer = value.boneName;
+	}
+	else buffer = "null";
+	
+	return buffer;
+}
+
+string NameToStringT2( TEntryType2& value )
+{
+    string buffer;
+	if (value.hasName > 0)
+	{
+		buffer = value.boneName;
+	}
+	else buffer = "null";
+	
+	return buffer;
+}
+
+typedef struct
+{
+    SetRandomBackColor();
+
+    TChunkHeader ChunkHeader;
+    TChunkTypePhysicsDictionaryHeader Header;
+    TEntryType1Wrapper Type1Entries(Header.entry1count)<name = "Entry Type 1 List">;
```

Comment:
Wrapper types aren't necessary, use an array of the type instead eg: TEntryType1 TypeEntries[Header.Entry1Count];

Reply:
Wrapper isn't necessary, but without the wrapper the arrays are not categorized into its own tab which leads to very poor visual clarity within the template view.
![image](https://user-images.githubusercontent.com/16394873/104111007-71610100-52ab-11eb-8d50-5504bb0e6c5a.png)